src/extensions/custom-view/bootstrap-table-custom-view.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @author: Dustin Utecht
```

Comment:
Add your email / personal website / github url. This is an amazing work :)

Reply:
done :-)